osmaxx/job_progress/middleware.py

```diff
@@ -39,6 +42,14 @@ def update_exports_of_request_user(request):
     for export in pending_exports:
         try:
             update_export_if_stale(export, request=request)
+        except HTTPError as e:
+            if hasattr(e.response, 'status_code'):
+                if e.response.status_code == requests.codes['not_found']:
+                    logger.exception("Export #%s doesn't exist on the conversion service anymore.", export.id)
```

Comment:
"anymore"? It might never have existed, depending on how the inconsistent state occurred. Case in point: Current state of our staging server.